app/uk/gov/hmrc/nisp/views/statepension.scala.html

```diff
@@ -95,124 +82,141 @@ <h2>@messages("nisp.nirecord.helpline.getHelp")</h2>
         userLoggedIn = true,
         showUrBanner =  true,
         browserTitle = Some(messages("nisp.main.title")),
-        pageInfo = Some(user.name.getOrElse("")),
         printableDocument = true,
-        pageTitle = Some(messages("nisp.main.h1.title")),
-        h1Class = Some("titleWithPageInfo"),
-        sidebarLinks = Some(sidebar),
-        articleClasses = Some("mainpage")
+        articleClasses = Some("mainpage fullwidth")
     )) {
 
 
-    <details id="nino">
-        <summary data-journey-click="accordion - click:Your State Pension summary:Show your National Insurance number"><span class="summary">@messages("nisp.show.nino")</span></summary>
-        <div class="panel">
-            <p class="bold">@user.nino</p>
-        </div>
-    </details>
-
-    <div class="highlighted-event govuk-box-highlight">
-        <p>@messages("nisp.main.basedOn") <span class="nowrap">@langUtils.Dates.formatDate(statePension.pensionDate). </span>
-           <span class="nowrap">@messages("nisp.main.caveats")
-            @messages("nisp.is")</span>
-        </p>
-        <p><em>@NispMoney.pounds(statePension.amounts.forecast.weeklyAmount) @messages("nisp.main.week")</em></p>
-        <p> @NispMoney.pounds(statePension.amounts.forecast.monthlyAmount) @messages("nisp.main.month"),
-            @NispMoney.pounds(statePension.amounts.forecast.annualAmount) @messages("nisp.main.year")
-        </p>
-    </div>
-
-    @if(statePension.mqpScenario.isDefined)  {
-
-        <p>@messages("nisp.main.caveats")</p>
-        <ul class="list-bullet">
-            <li>@Html(messages("nisp.main.notAGuarantee"))</li>
-            <li>@Html(messages("nisp.main.isBased", langUtils.Dates.formatDate(statePension.earningsIncludedUpTo)))</li>
-            <li>@Html(messages("nisp.mqp.howManyToContribute",Time.years(statePension.amounts.forecast.yearsToWork)))</li>
-            <li>@Html(messages("nisp.main.inflation"))</li>
-            @if(statePension.pensionSharingOrder) {<li>@Html(messages("nisp.main.psod"))</li>}
-        </ul>
-        @if(statePension.numberOfQualifyingYears > 0) {
-            <p>@messages("nisp.mqp.youCurrentlyHave", Time.years(statePension.numberOfQualifyingYears),Constants.minimumQualifyingYearsNSP)</p>
-        } else {
-            <p>@messages("nisp.mqp.youCurrentlyHaveZero", Constants.minimumQualifyingYearsNSP)
-        }
-
-        @if(!isAbroad) {
-            <div class="panel-indent">
-                <p>@Html(messages("nisp.mqp.overseas"))</p>
+
+    <div class="grid-row">
+        <div class="column-two-thirds">
+            <div class="page-info">@user.name.getOrElse("")</div>
+            <h1 class="titleWithPageInfo">@messages("nisp.main.h1.title")</h1>
+            <details id="nino">
+                <summary data-journey-click="accordion - click:Your State Pension summary:Show your National Insurance number"><span class="summary">@messages("nisp.show.nino")</span></summary>
```

Comment:
Seems a bit odd to use details pattern to just reveal a NINO. A design decision, but I'd be in favour of just showing this without hiding it away in a details. (This comment is made without viewing this example in the browser)

Reply:
it came from business a while back as nino is classed sensitive/personal information so they didnt wanted to show it by default

Reply:
👍